docs/installation/manage/_index.md

```diff
@@ -109,50 +140,71 @@ brew services restart viam-server
 {{% /tab %}}
 {{< /tabs >}}
 
-## Troubleshooting
+## View `viam-server` logs
 
-{{< tabs name="Troubleshooting">}}
-{{% tab name="Linux" %}}
+`viam-server` writes log messages as it starts up and runs, providing useful information when managing or troubleshooting the process.
+Use the following commands to view these log messages locally on your system.
 
-### View Logs
+{{< alert title="Tip" color="tip" >}}
+If your system is able to connect with the Viam app, logs can also be viewed in the **Logs** tab on [the Viam app](https://app.viam.com/).
+{{< /alert >}}
+
+Select the tab below for your platform:
+
+{{< tabs name="View logs">}}
+{{% tab name="Linux"%}}
+
+### As a system service
+
+If you are running `viam-server` as a system service, run the following command to view log messages:
 
 ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
 sudo journalctl --unit=viam-server
 ```
 
-If the robot is able to connect with the Viam app, logs can also be viewed in the **Logs** tab on the [Viam app](https://app.viam.com/).
-
-### SquashFS Errors
+Use the arrow keys to page vertically or horizontally through the log messages.
 
-Look like this...
+You can also view log messages specific to `viam-server` in the `syslog` with the following command:
 
-```sh {id="terminal-prompt" class="command-line" data-prompt="$" data-output="1-10"}
-Feb 10 13:11:26 hydro3-pi viam-server[933]: Something went wrong trying to read the squashfs image.
-Feb 10 13:11:26 hydro3-pi viam-server[933]: open dir error: No such file or directory
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+grep viam-server /var/log/syslog

```

Comment:
Q: This might only be correct for Debian-based distros. Do we pipe to the local logging facility - in which case this would be `/var/log/messages` on RHEL&friends, for example?

Maybe I should remove this alternate option, here and on line 177-181. LMK.

Reply:
Yeah, and only then if syslog forwarding is enabled in journald's config.

We do no explicit system logging, we rely on systemd to do that. (That'll change with the agent I'm working on>) Other distros without  systemd aren't even supported with our install script.